src/main/resources/web/app/qs-guide.ts

```diff
@@ -102,6 +102,7 @@ export class QsGuide extends LitElement {
   @property({type: String}) keywords: string;
   @property({type: String}) content: string | [string]
   @property({type: String}) origin: string = "quarkus";
+  @property({type: String, attribute: 'origins-with-relative-urls'}) originsWithRelativeUrls: string[] = [];
```

Comment:
I imagined passing a URL here, so that we can possibly handle multi-language search in the future, but... I see how this is probably better with the current code.
LGTM, thanks!